### ISSUE TITLE
Remove alt text from search results, cards, and item view images

### ISSUE THREAD

**Issue by tpendragon** [pulibrary/dpul-collections#824]
### Summary or User Story

Our images don't have useful alt text right now, and "image 1" just sounds bad. All of our images from the repository should have `alt=""` - none is better than bad, when it comes to reading the page. Hopefully in many cases there's enough information on the page, and if there's not then we need a feature for creating alt for images anyways.

### Acceptance Criteria

- [ ] Images without useful alt descriptions have an empty alt tag.

### First step

Search for where we're generating alt tags like "Image 1" and remove them.
